torch/testing/_internal/common_methods_invocations.py

```diff
@@ -234,6 +234,8 @@ def __init__(self,
                  gradcheck_fast_mode=None,  # Whether to use the fast implmentation for gradcheck/gradgradcheck.
                                             # When set to None, defers to the default value provided by the wrapper
                                             # function around gradcheck (testing._internal.common_utils.gradcheck)
+                 inplace_variant=_NOTHING,  # explicitly pass the inplace variant of the operator if required
```

Comment:
Using `_NOTHING` so that user can choose to pass `None`.

Reply:
Nice extension of the OpInfo architecture